elements/pfe-sass/maps/_typography.scss

```diff
@@ -81,3 +81,148 @@ $pf-type-sizing--titles: (
   title--m-sm--FontSize:    #{pfe-var(FontSize--sm ) },   // 14px
   title--m-xs--FontSize:    #{pfe-var(FontSize--xs ) },   // 12px
 );
+
+$pf-typography-sets: (
+  title: (
+    "6xl": (
```

Comment:
The quoting is important here since it starts with a number.  If Sass sees the number first, it assumes type = Number and for some reason ignores the "xl" bit.  If it's quoted, all is good.